### PR TITLE
Auth0 Updated Documentation 

### PR DESCRIPTION
**Pull request by nicolas-ethyca**
### Description of Changes

* [x] Screenshots and steps to configure Auth0

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* [x] Issue Requirements are Met
* [x] Relevant Follow-Up Issues Created

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
@nicolas-ethyca I can review this when its ready, but I'm not sure what to do about the failures

File: pages/fides/dsr_quickstart/saas_connectors/example_configs/auth0.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+
+# Auth0
+
+## Implementation Summary
+
+Fides uses the following Auth0 endpoints to retrieve and delete Personally Identifiable Information (PII) when processing a Privacy Request. Right to Access and Right to Delete (Right to Forget) support for each endpoint is noted below.
```
Comment (ThomasLaPiana) [inline pages/fides/dsr_quickstart/saas_connectors/example_configs/auth0.md:6]:
```suggestion
Fides uses the following Auth0 endpoints to retrieve and delete Personally Identifiable Information (PII) when processing a Privacy Request. Data access and erasure support for each endpoint is noted below.
```

I think this is technically a more accurate term

**Review by ThomasLaPiana — APPROVED** [review]

**Comment by nicolas-ethyca** [comment]
May I hit merge @suchiroll

**Comment by nicolas-ethyca** [comment]
Redone in https://github.com/ethyca/fidesdocs/pull/209

**Pull request closed without merging**